### DISCUSSION TITLE
Minecraft mod crashes when i try to launch it in vscode

### DISCUSSION THREAD

**Discussion by BjornSteinar** [FabricMC/fabric-api#1977] [Mod Dev Support] [answered]
I just added an item to the mod and there are no problems shown in vscode but when i try to launch minecraft it just crashes and i cant find the problem
[crash-2022-01-29_20.33.53-client.txt](https://github.com/FabricMC/fabric/files/7964696/crash-2022-01-29_20.33.53-client.txt)
:

**Comment by Juuxel** [comment, marked as answer]
The problem is that `Identifier`s can only contain lowercase characters (together with numbers and `_.-`) and you're trying to use uppercase ones:

> Non [a-z0-9_.-] character in namespace of location: EggertSteinarVignisson:test

**Reply by BjornSteinar** [reply]
thank you it works now
